src/main/java/edu/ucla/library/iiif/fester/handlers/GetCollectionHandler.java

```diff
@@ -52,8 +52,8 @@ public void handle(final RoutingContext aContext) {
             } else {
                 final Throwable aThrowable = send.cause();
                 final String exceptionMessage = aThrowable.getMessage();
-                final String errorMessage = LOGGER.getMessage(MessageCodes.MFS_009, collectionName,
-                        HTTP.INTERNAL_SERVER_ERROR, exceptionMessage);
+                final String errorMessage = LOGGER.getMessage(MessageCodes.MFS_009, collectionName, HTTP.NOT_FOUND,
```

Comment:
So, I see we have an inconsistency here, but I'm not sure changing the exception message to a 404 accurately captures what's happened. Looking at our S3BucketVerticle, the `get` basically just checks for `get.statusCode() == HTTP.OK` and, if it doesn't find that, fails the response. That failure though doesn't really make a distinction between a failed because it's a 404 and failed because there was some sort of other failure (500, etc.) I'm wondering if the fail int code from the S3BucketVerticle shouldn't be the HTTP response code instead of the MessageCodes' int code (which is kind of useless since that particular message code's value is just `{}`). It seems like that failure int code should be actionable if we can set it.